setup.py

```diff
@@ -22,9 +22,9 @@
 
 project_name = 'jax'
 
-_current_jaxlib_version = '0.4.28'
+_current_jaxlib_version = '0.4.29'
```

Comment:
This shouldn't be bumped.

Reply:
only `_latest_jaxlib_version_on_pypi`

Reply:
When does that get bumped?

Reply:
When you start the release. It's the first step.